modules/cnf-performing-end-to-end-tests-for-platform-verification.adoc

```diff
@@ -721,25 +835,115 @@ More histogram entries ...
 # Thread 8: 45766 56169 56171 ...# 00015 others
 # Thread 9: 02974 08094 13214 ... # 00090 others
 ----
-<1> The docker arguments provided by the user.
+<1> The `podman` arguments provided by the user.
 <2> The user is notified about the measured latency and the configured latency.
 <3> The arguments for the `cyclictest` command.
 <4> The maximum latencies measured on each thread.
 
+[id="cnf-performing-end-to-end-tests-capturing-results-cyclictest"]
+==== Capturing the results
+
+The same results can indicate different results for different workloads. For example, spikes up to 18μs is acceptable for 4G DU workloads but not for 5G DU workloads. Spikes above 20μs are not acceptable in any case.
```

Comment:
The same output, maybe too many iterations of results

Reply:
I'm not sure I understand this comment. Could you please elaborate/rephrase?

Reply:
I see! Thanks for the clarification, I'll update it